.test-infra/kubernetes/kafka-cluster/03-zookeeper/30service.yml

```diff
@@ -23,4 +23,3 @@ spec:
     name: client
   selector:
     app: zookeeper
-  type: LoadBalancer
```

Comment:
Is this the change that hides the client port?

Reply:
Yes. Default value is ClusterIP which only assigns a cluster-private ip address.